docs/jenkins.md

```diff
@@ -0,0 +1,272 @@
+* Go to __Manage Jenkins > Manage Credentials__
+* Next to (Global), select __Add credentials__
+
+    ![Add Credentials](assets/add_credentials.png)
+    
+* For __Kind__, select __Secret Text__
+* Enter the following information:
+    * Scope: Global
+    * Secret: 'your-sauce-username'
+    * ID: 'sauce-username'
+    * Description: Sauce Labs Username
+* Repeat the above steps for your Sauce Labs Access Key
+
+    ![Secrets](assets/secrets.png)
+
+    > For further information on how to store your Sauce Labs credentials in Jenkins, visit [the Jenkinsfile documentation](https://www.jenkins.io/doc/book/pipeline/jenkinsfile/#handling-credentials).
+
+## Create the `saucectl` Configuration
+
+Create the `.sauce` directory at the root of your project and add a `config.yaml` file that points to your existing `tests` directory. 
+
+With the `suites` field you can specify a group of tests as well as the browser `settings` you wish to use.
+
+Below are some examples:
+
+<!--DOCUSAURUS_CODE_TABS-->
+<!--puppeteer-->
+
+```sh
+apiVersion: v1alpha
+metadata:
+  name: Testing Puppeteer Support
+  tags:
+    - e2e
+    - release team
+    - other tag
+  build: Release $CI_COMMIT_SHORT_SHA
+files:
+  - ./tests
+suites:
+  - name: "chrome"
+    match: ".*.(spec|test).js$"
+    settings:
+      browserName: "chrome"
+image:
+  base: saucelabs/stt-puppeteer-jest-node
+  version: v0.1.8
+sauce:
+  region: us-west-1
+```
+
+<!--playwright-->
+
+```sh
+apiVersion: v1alpha
+metadata:
+  name: Testing Playwright Support
+  tags:
+    - e2e
+    - release team
+    - other tag
+  build: Release $CI_COMMIT_SHORT_SHA
+files:
+  - ./tests
+suites:
+  - name: "chrome"
+    match: ".*.(spec|test).js$"
+    settings:
+      browserName: "chrome"
+image:
+  base: saucelabs/stt-playwright-jest-node
+  version: v0.1.9
+sauce:
+  region: us-west-1
+```
+
+<!--testcafe-->
+
+```sh
+apiVersion: v1alpha
+metadata:
+  name: Testing TestCafe Support
+  tags:
+    - e2e
+    - release team
+    - other tag
+  build: Release $CI_COMMIT_SHORT_SHA
+files:
+  - ./tests
+suites:
+  - name: "chrome"
+    match: ".*.(spec|test).js$"
+    settings:
+      browserName: "chrome"
+image:
+  base: saucelabs/stt-testcafe-node
+  version: v0.1.7
+sauce:
+  region: us-west-1
+```
+
+<!--cypress-->
+
+```sh
+apiVersion: v1alpha
+metadata:
+  name: Testing Cypress Support
+  tags:
+    - e2e
+    - release team
+    - other tag
+  build: Release $CI_COMMIT_SHORT_SHA
+files:
+  - ./tests
+suites:
+  - name: "chrome"
+    match: ".*.(spec|test).js$"
+    settings:
+      browserName: "chrome"
+image:
+  base: saucelabs/stt-cypress-mocha-node
+  version: v0.1.11
+sauce:
+  region: us-west-1
+```
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+## Configure the Jenkins Pipeline
+
+Add the `Jenkinsfile` at the root of your project directory so that Jenkins can detect changes and run `saucectl` accordingly.
+In the examples below we've set our `environment` variables to the secrets that we configured in Jenkins:
```

Comment:
I know I already said this in the other file, but I would stay away from using "we" in this context.